docs/index.md

```diff
@@ -0,0 +1,19 @@
+Documentation
```

Comment:
I wonder, is there a auto tool for this?

Reply:
We should plan to push the docs directory to github.io.  I'm not sure whether you are supposed to use index.md or README.md when you do that.

Reply:
@nciric Not familiar, though what I did was rather mechanical: inserted the first paragraph of each document into a .md table.

@sffc sgtm; until that time, perhaps this works.

Any other actionable feedback?